### ISSUE TITLE
Smartto Tool: Linux version - Debian

### ISSUE THREAD

**Issue by kaktusus** [Geeetech3D/Smartto-IAR#1]
Hi :wink: 

Is it possible to create a version of these tools for Linux? Especially the installation package for the Debian distribution.

**Comment by Geeetech3D** [comment]
@kaktusus 
Yep, but people often use *nix without X Window. For easier to upgrade the machine, we will provide a command line script for Linux user.
